depsolver.js

```diff
@@ -6,6 +7,7 @@ const fs = require('fs');
 function resolvePath(file, x) {
   if (x.startsWith('@rpath')) {
     let baseIndex = file.lastIndexOf('/Frameworks');
+    let basePath = '/';
```

Comment:
Absolute paths are ignored because those point to system libraries which we are not going to resign. The /Frameworks match is needed to resolve @rpath because it is possible to have multiple nested frameworks embedded inside the ipa. This worked for me but i bet there are smarter ways to solve that, probably needed to RE the ios dyld to understand in depth